.env.j2

```diff
@@ -0,0 +1,23 @@
+# Sets the location of the flask application object: https://flask.palletsprojects.com/en/2.3.x/cli/#application-discovery
+# Needed because of the presence of the "todo_app" folder
+FLASK_APP=todo_app/app
+
+# Turn on debug mode (which enables reloading on code changes and the interactive debugger: https://flask.palletsprojects.com/en/2.3.x/config/#DEBUG)
+FLASK_DEBUG=true
+
+# Required to use Flask's session feature (for storing todos in a browser cookie in the first exercise): https://flask.palletsprojects.com/en/2.3.x/config/#SECRET_KEY
+SECRET_KEY=secret-key
+
+# Trello token and API key for utilising Trello REST API
+{% block content %}
+TRELLO_API_KEY={{trello_api_key}}
+TRELLO_TOKEN={{trello_token}}
+{% endblock %}
+
+# Trello Board ID
+TRELLO_BOARD_ID=64f1ece8f91c50af071d08d6
+#64e8b10e8ca37a0e9f16289f
+# Trello To Do List ID
+TRELLO_TODO_LIST_ID=64f596a2044f5b0dbabd927e
+# Trello Done List ID
+TRELLO_DONE_LIST_ID=64f596a248e42ea1b88a70f0
```

Comment:
If this were a real project, we'd likely want to swap these per environment, so we might mark these as variables too, but we could for example set give them these defaults in the `vars_prompts` section since they're not sensitive